src/ui.rs

```diff
@@ -1055,7 +1055,7 @@ fn show_about() -> Result<(), Error> {
 pub fn display_error(result: Result<(), Error>) {
     #[cfg(not(test))]
     if let Err(e) = result {
-        if let Some(g_error) = e.downcast_ref::<glib::Error>() {
+        if let Some(g_error) = e.downcast_ref::<gkt::glib::Error>() {
```

Comment:
Typo here.

Reply:
bah, I was sure I had built that, evidently not. Thanks for the fix.